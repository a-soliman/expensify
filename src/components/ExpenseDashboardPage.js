import React from 'react';
import ExpensesSummery from './ExpensesSummery';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';

const ExpenseDashboardPage = () => (
    <div>
        <ExpensesSummery />
        <ExpenseListFilters />
        <ExpenseList />
    </div>
);

export default ExpenseDashboardPage;