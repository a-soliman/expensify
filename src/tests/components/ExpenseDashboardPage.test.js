import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';

describe('ExpenseDashboardComponent', () => {
    test('Should render ExpenseDashboard Component', () => {
        const wrapper = shallow(<ExpenseDashboardPage />);
        expect(wrapper).toMatchSnapshot();
    });
});