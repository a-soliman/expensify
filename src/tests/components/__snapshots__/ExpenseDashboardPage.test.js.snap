// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpenseDashboardComponent Should render ExpenseDashboard Component 1`] = `
<div>
  <Connect(ExpensesSummery) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
